lab-beeksma/.eslintrc

```diff
@@ -0,0 +1,21 @@
+{
+  "rules": {
+    "no-console": "off",
+    "indent": [ "error", 2 ],
+    "quotes": [ "error", "single" ],
+    "semi": ["error", "always"],
+    "linebreak-style": [ "error", "windows" ]
```

Comment:
You have two options to fix the build.

1. Remove this line altogether.
2. Switch it back to `unix` and add a `.gitattributes` that includes `*.js text eol=LF`, which forces Git to checkout with LF instead of CRLF.